content/en/security/cloud_security_management/agentless_scanning/deployment_methods.md

```diff
@@ -17,10 +17,11 @@ further_reading:
 
 There are two recommended ways to deploy Agentless scanners in your environment, either using cross-account scanning, or same account scanning.
 
-**Note**: When using Agentless Scanning, there are additional costs for running scanners in your cloud environments. To optimize on costs while being able to reliably scan every 12 hours, Datadog recommends setting up Agentless Scanning with Terraform as the default template, as this also avoids cross-region networking. 
 
 To establish estimates on scanner costs, reach out to your [Datadog Customer Success Manager][1].
 
+The scanner cost is under $1 per scanned host per year for accounts following the [recommended configuration](#recommended-configuration).
```

Comment:
Sorry I missed this. Is this guaranteed to be under $1? If there's a chance that this could change in the future, recommend leaving out specific pricing language.

```suggestion
To learn more about cost efficiency, see the [Recommended configuration](#recommended-configuration).
```

Reply:
That was the whole purpose of this update: be more transparent on the pricing customers will face. That's something we're actively looking at reducing and that we've been pessimistic about here.